views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js

```diff
@@ -103,6 +103,7 @@ define([
             var $choiceBox = $(this);
             var state;
             var eliminator = e.target.dataset && e.target.dataset.eliminable;
+            var input = this.querySelector('.real-label > input');
```

Comment:
Why don't you use jQuery to access the DOM? Is there a particular reason?
I am asking that because the other code in the same module is using jQuery